### PR TITLE
Dodge name clash

### PR DESCRIPTION
**Pull request by mandolaerik**
windows.h defines a macro 'interface'.

Reported by Li Wang and Liang Yi

### CODE REVIEW

**Comment by mandolaerik** [comment]
[Test W](https://teamcity01-ir.devtools.intel.com/viewQueued.html?itemId=10909600)

**Comment by syssimics** [comment]
Verification [#10909571](https://teamcity01-ir.devtools.intel.com/viewLog.html?buildId=10909571): pass

**Comment by syssimics** [comment]
Verification [#10910025](https://teamcity01-ir.devtools.intel.com/viewLog.html?buildId=10910025): pass

File: test/tests.py [hunk 1]
```diff
@@ -1444,28 +1440,6 @@ def validate_file(self, path, regexp):
...
-                        line = re.sub(
-                            r'(.*© \d\d\d\d-)(\d\d\d\d)(.*)$'.encode('utf-8'),
-                            replace_cr_text, line)
-                        result.append(line)
-                        if line != org_line:
-                            updated = True
-                if updated:
-                    with open(path, 'wb') as f:
-                        for line in result:
-                            f.write(line)
```
Comment (lwaern-intel) [inline test/tests.py:1468]: Deserves a dedicated commit.

Comment (mandolaerik) [reply test/tests.py:1468]: thanks, fixed

**Review by lwaern-intel — APPROVED** [review]
The fact a single extra line to `cident_renames` pulls all the necessary work is _really_ nice.
